netty/src/main/java/io/grpc/netty/GrpcSslContexts.java

```diff
@@ -84,6 +93,22 @@ private GrpcSslContexts() {}
       SelectedListenerFailureBehavior.ACCEPT,
       NEXT_PROTOCOL_VERSIONS);
 
+  private static final String SUN_PROVIDER_NAME = "SunJSSE";
+  private static final Method IS_CONSCRYPT_PROVIDER;
+
+  static {
```

Comment:
Should this be wrapped in a function to provide an easier to understand stack trace if this ends up throwing?

Reply:
I could, although I find it more confusing to have static methods that are called _before_ static initialization completes. It's necessary and helpful at times, but when unnecessary I try to avoid it as I find it to be bug-prone.